test/errorshow.jl

```diff
@@ -535,6 +535,12 @@ end
     @test occursin("g28442", output[3])
     @test output[4][1:4] == " [2]"
     @test occursin("f28442", output[4])
-    @test occursin("the last 2 lines are repeated 5000 more times", output[5])
-    @test output[6][1:8] == " [10003]"
+    # Issue #30233
+    if Sys.isfreebsd() && ccall(:jl_is_debugbuild, Cint, ()) == 0
```

Comment:
Oh, I guess this doesn't properly reflect what the CI system is using; this branch is being hit on CI but the tests work there, so `@test_broken` fails.

Reply:
Another option is just to disable these tests entirely on FreeBSD. If what I wrote on #30233 is right, the tests only work due to a libunwind heuristic. So it seems like changing *any* compilation option on FreeBSD could potentially make these tests fail.